app/views/environments/welcome.html.erb

```diff
@@ -1,2 +1,5 @@
-<% content_for(:title, _("Enviroments")) %>
```

Comment:
This is the page title, it should still remain - but changed to Puppet Environments as well